### PR TITLE
fix octree max-packet-per-second setting

### PR DESCRIPTION
**Pull request by sethalves**
The value given to an OctreeQuery is a per-query value and shouldn't be saved as a perference.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 83748096efc79b858e6f52cec344770aba54811b available at: https://gist.github.com/f577aecaaf4c31c1ccda<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.dmg

File: libraries/octree/src/OctreeQuery.cpp [hunk 1]
```diff
@@ -11,22 +11,17 @@
...
-    _maxOctreePPS = maxOctreePacketsPerSecond.get();
+    _maxQueryPPS = DEFAULT_MAX_OCTREE_PPS;
 }
 
-void OctreeQuery::setMaxOctreePacketsPerSecond(int maxOctreePPS) {
-    if (maxOctreePPS != _maxOctreePPS) {
-        _maxOctreePPS = maxOctreePPS;
-        maxOctreePacketsPerSecond.set(_maxOctreePPS);
-    }
+void OctreeQuery::setMaxQueryPacketsPerSecond(int maxQueryPPS) {
```
Comment (ZappoMan) [inline libraries/octree/src/OctreeQuery.cpp]: can't you make this inline now?

**Comment by hifi-gustavo** [comment]
Build console output for commit: de9e74a725f96342c1971d6538868fae7c20605d available at: https://gist.github.com/f577aecaaf4c31c1ccda<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9bd101ee2ae3381d9b458fc8281083bda55461aa available at: https://gist.github.com/f577aecaaf4c31c1ccda<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ead200b77aa558b6d4bbd73a6f6f4e3e3efb9b0d available at: https://gist.github.com/f577aecaaf4c31c1ccda<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 635f49258d18a5932b5fdd45ab2e671d7f932abe available at: https://gist.github.com/f577aecaaf4c31c1ccda<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ddb907ae5fd7e83e6e8773595cd4416f173553d8 available at: https://gist.github.com/f577aecaaf4c31c1ccda<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.dmg

**Comment by ZappoMan** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: ddb907ae5fd7e83e6e8773595cd4416f173553d8 available at: https://gist.github.com/f577aecaaf4c31c1ccda<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4827.dmg

**Pull request merged**